### PR TITLE
[Hyperdrive] Revisiting Learning section

### PR DESCRIPTION
**Pull request by Maddy-Cloudflare**

### CODE REVIEW

File: content/hyperdrive/learning/connect-to-postgres.md [hunk 1]
```diff
@@ -137,24 +130,24 @@ export default {
...
 
-In cases where you need to issue these unsupported statements from your application, Cloudflare recommends setting up a second, direct client without Hyperdrive.
+In cases where you need to issue these unsupported statements from your application, the Hyperdrive team recommends setting up a second, direct client without Hyperdrive.
 
 ## Identify connections from Hyperdrive
 
 To identify active connections to your Postgres database server from Hyperdrive:
 
 - Hyperdrive's connections to your database will show up with `Cloudflare Hyperdrive` as the `application_name` in the `pg_stat_activity` table.
-- You can run `SELECT DISTINCT usename, application_name FROM pg_stat_activity WHERE application_name = "Cloudflare Hyperdrive"` to show whether Hyperdrive is currently holding a connection (or connections) open to your database.
```
Comment (Maddy-Cloudflare) [inline content/hyperdrive/learning/connect-to-postgres.md:157]: @elithrar was `usename` intentional here?

File: content/hyperdrive/learning/connect-to-postgres.md [hunk 2]
```diff
@@ -27,23 +25,23 @@ When using wrangler, replace the placeholder value provided to `--connection-str
...
 
-Refer to the [examples documentation](/hyperdrive/examples/) for step-by-step guides on how to set up Hyperdrive with several popular database providers.
+Refer to the [Examples documentation](/hyperdrive/examples/) for step-by-step guides on how to set up Hyperdrive with several popular database providers.
 
 ## Supported drivers
 
 Hyperdrive uses Workers [TCP socket support](/workers/runtime-apis/tcp-sockets/#connect) to support TCP connections to databases. The following table lists the supported database drivers and the minimum version that works with Hyperdrive:
 
-| Driver               | Docs                                 | Minimum Version Required |
+| Driver               | Documentation                                 | Minimum Version Required |
```
Comment (pedrosousa) [inline content/hyperdrive/learning/connect-to-postgres.md]:
```suggestion
| Driver               | Documentation                        | Minimum Version Required |
```

File: content/hyperdrive/learning/how-hyperdrive-works.md [hunk 3]
```diff
@@ -8,36 +8,46 @@ weight: 2
...
-- When that transaction has completed, the connection is returned to the pool.
-- Named prepared statements (which are rarely used), `SET` commands and advisory locks are not supported.
+The connection pool operates in transaction mode, where the client that executes the query communicates through a single connection for the duration of a transaction.
 
-Users can use features like `LOCAL` and `pg_advisory_xact_lock` which are scoped to single transactions.
+When that transaction has completed, the connection is returned to the pool.
 
-In cases where you need to issue these unsupported statements from your application, we recommend setting up a second, direct client without Hyperdrive.
+{{<Aside type="warning">}} 
+Named prepared statements (which are rarely used), `SET` commands and advisory locks are not supported. Use features such as `LOCAL` and `pg_advisory_xact_lock` which are scoped to single transactions. In cases where you need to issue these unsupported statements from your application, the Hyperdrive team recommends setting up a second, direct client without Hyperdrive.
```
Comment (pedrosousa) [inline content/hyperdrive/learning/how-hyperdrive-works.md]:
```suggestion
Named prepared statements (which are rarely used), `SET` commands, and advisory locks are not supported. Use features such as `LOCAL` and `pg_advisory_xact_lock` which are scoped to single transactions. In cases where you need to issue these unsupported statements from your application, the Hyperdrive team recommends setting up a second, direct client without Hyperdrive.
```

File: content/hyperdrive/learning/query-caching.md [hunk 4]
```diff
@@ -6,13 +6,13 @@ weight: 4
...
+Hyperdrive automatically caches the most popular queries executed against your database, reducing the need to go back to your database (incurring latency and database load) for every query.
 
 ## What does Hyperdrive cache?
 
-Because Hyperdrive understands database protocols, it can understand whether a query is mutating (writes to the database) or non-mutating (read-only), allowing it to safely cache read queries (only).
+Because Hyperdrive uses database protocols, it can differentiate between a mutating query (a query that writes to the database) and a non-mutating query (a read-only query), allowing Hyperdrive to safely cache read-only queries.
 
-Note that Hyperdrive does not just attempt to determine the difference between a `SELECT` and an `INSERT`, but parses the database wire-protocol and uses it to concretely determine whether a query is read-only (non-mutating) or not
+Hyperdrive determines the difference between a `SELECT` and an `INSERT`, as well as parsing the database wire-protocol and using it to differentiate between a mutating or non-mutating query.
```
Comment (pedrosousa) [inline content/hyperdrive/learning/query-caching.md]:
```suggestion
Besides determining the difference between a `SELECT` and an `INSERT`, Hyperdrive also parses the database wire-protocol and uses it to differentiate between a mutating or non-mutating query.
```
I think this approach conveys the original meaning a bit better.

**Review by pedrosousa — APPROVED** [review]

File: content/hyperdrive/learning/connect-to-postgres.md (hunk 1 above)
Comment (elithrar) [reply content/hyperdrive/learning/connect-to-postgres.md:157]: Yes.

Comment (elithrar) [reply content/hyperdrive/learning/connect-to-postgres.md:157]: image

https://www.postgresql.org/docs/9.3/view-pg-user.html

**Pull request merged**